### PR TITLE
Simplify readme

### PR DESCRIPTION
**Pull request by awisniew90**
Simplify the readme to a basic description of the stack and how to get started. The other content has been moved over to the wiki here: https://github.com/OpenLiberty/application-stack/wiki

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,194 +19,62 @@
...
 
-    This will initialize an Open Liberty project and download the default starter app artifacts.
+    This will download the default starter app
 
-1. Once your project has been initialized, you can push your application to your OpenShift cluster for the first time using the following command:
+1. Push your app to your cluster
 
     ```shell
     odo push
+    odo watch
```
Comment (scottkurz) [inline README.md:58]: Did you really mean to document this sequence?   

I would usually suggest:
1. odo push
2. odo log -f   
(then wait for the test success message).

File: README.md [hunk 2]
```diff
@@ -19,194 +19,62 @@
...
 
-    Upon a first time `odo push` invocation, the following occurs:
-    - An Open Shift managed kubernetes pod is launched within the namespace created in step 3 above,
-    - A container based on the image referenced in the devfile is started in that pod
-    - odo syncs your local source code directory to that container. (under the `/projects` directory)
-    - The app is compiled and built within the container and liberty dev mode is started to host the app once built.
-    - A URL is activated and is usable to access the app
-
-1. Retrieve the URL for your app using the odo CLI:
+1. Retrieve the URL for you app
```
Comment (scottkurz) [inline README.md:61]: you=>your

File: README.md [hunk 3]
```diff
@@ -19,194 +19,62 @@
...
-For more details on the application-stack-intro application, check out https://github.com/OpenLiberty/application-stack-intro
-
-## Odo development operations
-
-### PUSH
-By default, `odo push`, in conjuction with the java-openliberty devfile, runs a liberty:dev mode in "hot test" mode, where unit tests and integration tests get automatically re-executed after each detected change. These test results are visible in the OpenShift container logs. To see the realtime contents of the container logs, use the odo CLI:
-```shell
-odo log
-```
+You can now begin developing your app! Changes will be detected and pushed automatically.
```
Comment (scottkurz) [inline README.md:67]: Maybe it should be:

1. odo push
2. odo log -f  # wait
3. odo url list 
4. hit browser/curl
5. odo watch... Changes will be detected..automatically

**Review by scottkurz — CHANGES_REQUESTED** [review]
At a high-level I like the simplification but maybe I'd just produce this first batch of detail a bit differently as in my comment.

File: README.md (hunk 1 above)
Comment (awisniew90) [reply README.md]: I don't think the starter has any tests to run.

File: README.md (hunk 3 above)
Comment (awisniew90) [reply README.md]: done - I left out the odo log step since we dont have any tests to run out of the box with the starter

**Review by scottkurz — APPROVED** [review]

**Pull request merged**